ha/setup/create_pacemaker_resources.py

```diff
@@ -124,7 +124,7 @@ def haproxy(cib_xml, push=False):
 def sspl(cib_xml, push=False):
     """Create sspl clone resource in pacemaker."""
     # Using sspl-ll service file according to the content of SSPL repo
-    cmd_sspl = f"pcs -f {cib_xml} resource create sspl-ll systemd:sspl-ll clone op monitor interval=30 --group monitor"
```

Comment:
@ajay-paratmandali @ajaysrivas why monitor group is deleted here? Just in previous PR @ajaysrivas asked to add sspl to the group? 
https://github.com/Seagate/cortx-ha/pull/283#pullrequestreview-578441498

Reply:
This was a quick fix to separate clone and group. Didn't want to change the flow, so group was removed and it will be added again after single node.